src/FeedbackService.Data.Provider.MsSql.Ef/Migrations/20220715140000_InitialMigration.cs

```diff
@@ -0,0 +1,20 @@
+﻿using LT.DigitalOffice.FeedbackService.Data.Provider.MsSql.Ef;
```

Comment:
May be this class not need now?

Reply:
for now it's not but it will be there for sure because there is dbcontext